src/transformers/convert_slow_tokenizer.py

```diff
@@ -286,7 +286,7 @@ def converted(self) -> Tokenizer:
             bos = self.original_tokenizer.bos_token
             bos_token_id = self.original_tokenizer.bos_token_id
             tokenizer.post_processor = processors.TemplateProcessing(
-                single=f"{bos}:0 $A:0",  # token_type_id is 2 for Funnel transformer
+                single=f"{bos}:0 $A:0",
```

Comment:
Not sure there's a reason why this is modified?

Reply:
I think this comment is a leftover from copying the FunnelConverter, because note that the template doesn't have `:2` token type id anywhere, which is the case here:

https://github.com/huggingface/transformers/blob/b6303bb091adc04b0183a00a0f9c82c86e8a5989/src/transformers/convert_slow_tokenizer.py#L189